folding-schemes/src/lib.rs

```diff
@@ -43,6 +43,8 @@ pub enum Error {
     IVCVerificationFail,
     #[error("zkIVC verification failed")]
     zkIVCVerificationFail,
+    #[error("Committed instance is expected to an incoming (fresh) instance")]
```

Comment:
maybe `[...] to *be* an incoming`?

Reply:
Fixed in https://github.com/privacy-scaling-explorations/sonobe/pull/157/commits/8047ccf79586630e09d3de69af75cb3be2ea9910 😄